### ISSUE TITLE
Location Search: order changes

### ISSUE THREAD

**Issue by joshi1983** [hhaccessibility/hhaccessibility.github.io#400]
Sorting by rating should sort the highest rating to the top.

Default sorting should be by accessibility rating.  When accessibility ratings are equal, sort by distance.  When both are equal, sort alphabetically by name.

The following features shouldn't change:
- sort by name should continue being in ascending order as in starting with 'A' at top.
- sort by distance should continue being in ascending order as in starting with '0' at top.

This was discussed in the November 10, 2017 status meeting.

**Comment by joshi1983** [comment]
This would be a change in PHP.  I think it'll be in LocationSearchController.php.

**Comment by joshi1983** [comment]
Balraj seems almost complete with this.  I think he just needs to make a pull request.

**Comment by joshi1983** [comment]
@balraj18 can you give some testing instructions?  Elaborate on how you'd see the changes by clicking different columns and seeing navigating from the home page to see the default sorted column.

**Comment by gillbalraj** [comment]
Testing instructions are as follow:

- Sign-In to your AccessLocator account and **rate** few location in a category say either  Shopping,    Restaurant or Healthcare etc.

- GO to that category and hit the tag **Name, Rating(%) and Distance** one by one.
-  Observe the change in behavior of rows & columns as per instructed in details given above
![capture](https://user-images.githubusercontent.com/26493273/33233270-2a8091fe-d1e1-11e7-9477-625366769d20.JPG)
.

**Comment by princeseth** [comment]
This is working as expected: 

1. By clicking on tag **Name** the result gets sorted alphabetically. 
2. On clicking tag **Rating** the location with the highest rating comes at the top, and rest of location sorted based on least distance.
3. On clicking tag **Distance** the location result gets sorted from least to maximum distance.

"Default sorting should be by accessibility rating. When accessibility ratings are equal, sort by distance. When both are equal, sort alphabetically by name." I have tested this both on desktop and mobile device, and it is working fine.

**Comment by joshi1983** [comment]
Thanks, Prince.  I'll close this as he found everything in the scope of this issue working.
